node/res/tnet-staking.full.json

```diff
@@ -0,0 +1,131 @@
470776fc12c5fcb16d1a63990a22aa8ad03d445fa81f0d13a27723bfcf8f3f64e780d325a55804fd65381c58b7b60a80841f2b4422a7ca373ff496efd520e14bcdde70abf76b5452fda435c0ffbd8b44ca829b7fbe72f5fc6951daa5980a4043e927d283e0d59cd680b545518a9386d2c7008f1f845ee00bda741b54f4076d787bb54b1369f0a845328856d550b60f781e3cd9b6ba357b074fe6e6f0bc14036dfeb4c13fd40e40e9e8bfebe0a9c2e21efcc53dd495961ab6e4e9227633d9e74162406fbbd1ca7e9322cebd0ca9862ca13aedc9aafe496b21a2b511273b41248d61215932b60f1bd60fed85f29eca5146a7127d745f57a6ed1a15512201beb2abd35b7b9d7a3bef9b3fea0eb96073aa4fe285283d9f74d4bf85393ea71c403a166842bdc366fe71c77198856ed5291783cb141271218d64bae8ae409f28249743d0e9e5592ad2bc98ee6e16b52c339925bc6239d8c04c1ce429729041ca04211bb5a75dd4313d0cd73b20fcc40804375b60fa1ef5cab80ef3940beeccfa2122bd690eade3a007713d523be80e299f6b2a00882c2c3eed82e657a3fc0ff99e2637562d1bd3ccb80f170d56ca86da86c04db17639e7513d0603680cfc0eb3eb849e7de4b9e4496284949dce1c47c996e24853b3716e60a4344423d6d194382f52a53e6230a645e61f357c1f27f23c4811584495e78ab9da542b9348c20f5e8a60230f48245e0ffd38062cfffa88c990a61466284f4b0fc9d218de9747ee3f24980e15903549a861c3327b42d6ff979bbb55dd356eefc1d2672a4f6c14913f11bdd3d5f011e3d213d974fcaf5eb7ac3f2a22011327106f0b3881a93fbcf956d84d825b7c89b4f2513bfd93024b5d53fa9c309d1621078e289e5da46b7d9ad9ec18a92549af5184a3167170a75ef996be42973bce7fcf8d2c9060b79425a7015a5a48602dab6ca9c051508af1ed799771ec8129b21fd12835cb1a837069c99c1fc26a0a06359597fb3de8fe5b0776926563101f5c4be54f4b98ca9279026928e1f0d77e4657c11302b6b5908dc092b8926ae7a2d67ea40328c3fb91d2e41984a5810ad80c8b4bc504b251b7e1c0bcba669ab83e3007368832607b04873b73f758e3ec69de4c4678487c2948e9ed2cf1b672e37e8c851ed880944283ebbdcb82aee60adae18b1d1c651c39d471305562f73a78a7b891affec0694109f4c46b4c77e229724bd4f475a01dedb71d77663a2d3270db5b84806c0f69a293115fe82f40b43dabb40cebae4753c69989da7d14856dee5e17e95615aac8eab85b1a493f88ceb11a6fb32410fe3ff4c7473190909c62c04dc02b12f4f2b7768f158ebfd80f1800f724e41fbbadd470f43c84240ca285b08658ac037a58ebcaffcc98c418870b42cd2a5deb88d0008abe8073cb002f7c0f27704009ba5952f2f0cb25f54c1e352f6debf24c29f9f263b5147a5cf87e63a0701117489ad2bc0465a6a815d2ef3860c7250dc83b16bc14d2b3843f907f548678228ec0d942a7191cc12ad61646c3a23eafc81809e5bd9e6f6a41349c15c9f4b2a3d81c8e3bf624314dee90840b0d9de423bb2c65975f33c3868135b3b342730844cd937b52487fcb86d797f7151516ab9190958b15391b93bff8eab770419cb156a8f442bb27364f3324ae2d0b33073be822a74899353ec7fb92e80ac1d42782b1586688344994e2c4e2b1e09427574be0c2561e15760e3f803c0df821b3e697c5ec9805a0f3935205c07d9b2052d597101efc245a3dd98fe12df3f6f45f12d191061c80029f41db3ae7b21bcaf1c16e8ba71bf0ce594b7ba0e226949e9b19e3bad8dbc7a7afd648d92940cb8eb4d0059aa9d13af6034224570a5a2a37a28bd1ec7441e6e76637ff86a2a3594f5938716984b9a493ced2c4d8f095269b86ac55e5317c4f6e2e6c931490097422ab537bc045e57b87a4124997ff18b067ef6b04990c80b93d75ce5c89ff17e870d42d0094e94fbd4a58a9a5f88289fdfc6a72307a6d851678e3310cc3bc742c56283b35b9bd40be8b2553b49e1dc2d18119be282ac70d9c4346131be2cf9a0617ad791586caba3c023344e5ce522db9332077cb19983ece764dcac8d0219f
```

Comment:
Whose keys are these? Can you make sure to store the private keys on 1password?

Reply:
Sure will update 1password.